data/hai/mercenaries.txt

```diff
@@ -0,0 +1,828 @@
+				goto fake
+			label knowsbars
+			`	You recognize the same bars that earned you betrayal from a member of your crew. Something about them must make them special and hugely valuable to those who know of it.`
+				goto fake
+			label engine
+			`	These are some fine last-generation atomic engines, a true beauty in your eyes. But admiring the engines doesn't help you find the item at all. Why would there be anything near the engines?`
+				goto caught
+			label fake
+			choice
+				`	(Take a heavily-modified alien looking pistol.)`
+				`	(Take a strong personal shield.)`
+				`	(Take what looks like a bomb.)`
+				`	(Drop some bars on the ground to take them later.)`
+					goto takebars
+			label wrongitem
+			`	You take the item, hoping it is the right one, and swiftly leave the ship. If only you had been given a more precise description, you could know for sure. Some pirates seem to notice you, and maybe they suspect something or recognize this item, but nobody tries to stop you. As you reach your ship, you notice that the Degenerate just launched, right before you. They must have noticed the item was missing.`
+				flee
+			label takebars
+			`	How many stacks of bars will you take? The more, the bigger the risk and reward...`
+			choice
+				`	(One will do.)`
+					goto one
+				`	(Three should be enough.)`
+					goto three
+				`	(Five.)`
+					goto five
+				`	(Eight could be reasonable.)`
+				`	(Take as much as I can: fifteen!.)`
+				`	(Never mind, this isn't a good idea.)`
+					goto fake
+			`	You got too greedy. As you're dropping the bars, you feel something cold hit your head. You never wake up again.`
+				die
+			label one
+			action
+				outfit "Gold Bars"
+			``
+				goto dropbars
+			label three
+			action
+				outfit "Gold Bars" 3
+			``
+				goto dropbars
+			label five
+			action
+				outfit "Gold Bars" 5
+			label dropbars
+			`	You drop the gold and then leave the ship. Afterwards, you go around the ship to the location where they are partially buried in the sand, and take them discreetly with some members of your crew. You have to silence one or two curious pirates, but that's about it, and you can now head back to your ship. As expected, they have noticed the theft. you see them launching before you can do so yourself. This will be a bumpy ride.`
+				flee
+			label caught
+			`	You do not manage to find the item here. However, you've attracted way too much suspicion by looking and opening crates everywhere you could. One local worker asks you something about mechanics. Being no engineer yourself, you cannot make any sense of what he says, even less formulate a correct response, and he calls a guard over you. You are captured and sold as a slave, condemned to a few years of hard work before a sad death.`
+				die
```

Comment:
~~Time flies when you are having fun.~~

Reply:
ik right them years passed like its nothing